File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        List<Integer> l=Arrays.stream(nums).boxed().toList();
        Set<Integer> s=new HashSet<>(l);
        int m=Integer.MIN_VALUE;
        int a[]=new int[s.size()];
        int c=0;
        for(int i:s)
        {
            int k=Collections.frequency(l,i);
            m=Math.max(k,m);
            a[c++]=k;
        }
        final int k=m;
        long l1=Arrays.stream(a).filter(x->x==k).count();
        return (int)l1*k;
    }
}

File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.c
int maxFrequencyElements(int* nums, int numsSize) {
    int n=numsSize;
    int i,j,a[n],b[n],c1=0,f,c,l;
    for(i=0;i<n;i++)
    {
        f=0;
        for(j=0;j<i;j++)
        {
            if(nums[i]==nums[j])
            {
                f=1;
                break;
            }
        }
        if(f==0)
        {
            b[c1]=nums[i];
            c=1;
            for(j=i+1;j<n;j++)
            {
                if(nums[i]==nums[j])
                {
                    c++;
                }
            }
            a[c1++]=c;

        }
    }
    l=a[0];
    for(i=0;i<c1;i++)
    {
        if(a[i]>l)
            l=a[i];
    }
    c=0;
    for(i=0;i<c1;i++)
    {
        if(l==a[i])
            c++;
    }
    int s=0;
    for(i=0;i<c;i++)
    {
        s=s+l;
    }
    return s;
}